hw_7_classes.py

```diff
@@ -0,0 +1,233 @@
+import json
+import pprint
+import os
+import pathlib
+import fnmatch
+
+
+# Задача-1
+# У вас есть список(list) IP адресов. Вам необходимо создать
+# класс который будет иметь методы:
+# 1) Получить список IP адресов
+# 2) Получить список IP адресов в развернутом виде
+# (10.11.12.13 -> 13.12.11.10)
+# 3) Получить список IP адресов без первых октетов
+# (10.11.12.13 -> 11.12.13)
+# 4) Получить список последних октетов IP адресов
+# (10.11.12.13 -> 13)
+
+
+class IpAddress:
+
+    ip_list = ['10.11.12.13', '192.168.12.1', '147.22.243.33']
+
+    def __init__(self):
+        self._list = IpAddress.ip_list
```

Comment:
If you have class attributes, it means they are shared between all the objects of this class. No need to define it here. However, I would use an instance attribute instead of a class one. So, it is possible to define different ip lists for different instances